docs/source/driver-framework/drivers-overview.rst

```diff
@@ -51,18 +49,18 @@ The diagram features several entities that comprise the platform and its connect
   message bus is built around existing message bus software; currently VOLTTRON supports RabbitMQ and ZeroMQ. The
   VOLTTRON integration includes Pub/Sub and JSON RPC interfaces for agent and driver communication.
 * VOLTTRON Platform Agents and Subsystems - These agents and subsystems are installed on the platform to manage the
-  platform. They provide many user facing functions, aid in communication and manage other agents and drivers.
-* User's Agents - These agents are either agents included in the core repository but installed by a user, or user built
+  platform. They provide many user facing functions, aid in communication, and manage other agents and drivers.
+* User's Agents - These agents are either agents included in the core repository but installed by a user or user-built
   agent modules. They may perform a huge variety of user specified tasks, including data collection, device control,
   simulation, etc.
 * Platform Driver Agent - This agent is installed by a user to facilitate communication with drivers. Drivers should not
   communicated with directly - the platform driver implements several features for communicating with drivers to ensure
   smooth operation and consistent driver behavior.
-* Actuator agent - This agent is installed by user to provide scheduling capability for controlling drivers. The
+* Actuator agent - This agent is installed by a user to provide scheduling capability for controlling drivers. The
```

Comment:
All agents are "installed" so perhaps modify those not to have This agent is installed...instead "this agent provides scheduling"....